setup.py

```diff
@@ -11,7 +11,10 @@
     install_requires=[
         'click',
         'pyyaml',
+        'pylint', 
         'jsonschema',
+        'cylc-flow',
+        'cylc-rose',
```

Comment:
@bcc2761 brought this up somewhere else. I think this is OK, but probably won't be used.

`pip install .` called from the meta.yaml won't install prerequisites (it sort of assumes the prerequisites can be installed through conda), but interactive fre-cli developers who `pip install .` outside of conda might take advantage of this

Reply:
i am probably that sole fre-cli dev... this is clearly not generally needed.

Reply:
> @bcc2761 brought this up somewhere else. I think this is OK, but probably won't be used.
> 
> `pip install .` called from the meta.yaml won't install prerequisites (it sort of assumes the prerequisites can be installed through conda), but interactive fre-cli developers who `pip install .` outside of conda might take advantage of this

Correct, this is why I had trouble integrating my 'gfdlfremake' pypi package.